extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -307,6 +448,26 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     );
   }
 
+  /**
+   * @return {string} ad client ID for the current ad unit.
+   */
+  getAdClientId() {
+    const adClientId = (
```

Comment:
let adClientId = (this.element.getAttribute('data-ad-client') || '').toLowerCase();
if (!/^ca-/i.test(adClientId)) adClientId = `ca-${adClientId}`;
return adClientId;

Reply:
done